xdsl/transforms/experimental/ConvertStencilToLLMLIR.py

```diff
@@ -414,3 +465,5 @@ def ConvertStencilToLLMLIR(ctx: MLContext, module: ModuleOp):
                                         apply_recursively=False,
                                         walk_reverse=True)
     the_one_pass.rewrite_module(module)
+    PatternRewriteWalker(LowerHaloExchangeToMpi(
```

Comment:
Why not add this to StencilConversion?

Reply:
Because then it doesn't work :( I tried

Reply:
Ok, won't block for this, but sounds like worth a check together 🙂